### PR TITLE
Please review my PR and suggests necessary changes.

### PR DESCRIPTION
**Pull request by paraggarg1102**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
 # Pole-Climbing-Robot
 Srishti 2020
+## A. Images of the project
```
Comment (SanjeevKrishnan) [inline README.md]: Remove this heading. Only Image has to be present without heading

File: README.md [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+![Final Product](https://github.com/DanielLarusso1/Pole-Climbing-Robot/blob/master/Images%20and%20Videos/Images/Robot%20CAD.jpeg)
+## B. Abstract
```
Comment (SanjeevKrishnan) [inline README.md]: Sub-Hadings should not be in list. Remove 'A.', 'B.', 'C.', ... from headings

File: README.md [hunk 3]
```diff
@@ -1,2 +1,62 @@
...
+### Pole climbing robot utilizes a rack and pinion system to perform vertical motion up and down a pole, with the caterpillar motion of the top plate and bottom plate. The robot can climb up the poles within a certain range of diameters.
```
Comment (SanjeevKrishnan) [inline README.md]: Content inside sub-heading should be normal text. Remove "###"

File: README.md [hunk 4]
```diff
@@ -1,2 +1,62 @@
...
+### Electricians and workmen have to climbup an electric pole in case of a malfunction, this leads to several accidents. This can be avoided if a robot can be made to climb the pole and perform specialised functions. As a first step towards that, this robot has been designed to climb a pole. It can also be used for surveillance and fruit harvesting.
+
+## D. Mechanical Aspect of the Design
+### The robot consists of a top plate is connected to the pinion and a clamp whereas the bottom plate is connected to the rack and the lower clamp. The clamp is 3d printed and is specifically designed for this purpose. It is padded with a soft gripping rubbery material. It's grip is loosened or tightened using the motion of a lead screw. The two plate system is stabilised by two rods which pass through the top plate through linear bearings and are connected to the bottom plate.
+
+## E. Electronics Aspect of the Design
+### The micro-controller used is Arduino UNO and Cytron motor drivers. The pinion is rotated by a DC motor with an encoder. The lead screw of the clamp is rotated by a DC motor with an encoder. 
+
+## F. Cost Structure (In Rupees)
+### 1. Arduino UNO		500
```
Comment (SanjeevKrishnan) [inline README.md]: Cost Structure has to be in tabular form

File: README.md [hunk 5]
```diff
@@ -1,2 +1,62 @@
...
+### 2. The rack and pinion is slightly slower than the faster actuators like hydraulic and pneumatic.
+### 3. The clamp design limits the width of the rod that can be climbed.
+### 4. The bot's weight increases because of the several actuators and other parts.
+
+## I. Future Improvement
+### 1. Design a prototype at big size, which can be used for pole with different diameters and high payloads capability
+### 2. Replace the grippers with a better gripping mechanism ( Ex: Soft robot gripper or more number of grippers to climb poles of any texture and diameter)
+### 3. Remotely control the robot from a smart device.
+
+## J. Team members(in alphabetical order)
```
Comment (SanjeevKrishnan) [inline README.md]: Remove "Alphabetical Order" from heading. Also hyperlink team members and mentors name with their GitHub profile link.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,62 @@
...
+### 1. Akshara Singh
+### 2. Nitika Gupta
+### 3. Parag Garg
+### 4. Sarthak Sharma
+
+## K. Mentors (in alphabetical order)
```
Comment (SanjeevKrishnan) [inline README.md]: Remove "in Alphabetical Order" from heading

File: README.md [hunk 7]
```diff
@@ -1,2 +1,62 @@
...
+### 1. Aditya Raj
+### 2. Rishika Chandra
+
+## L. Proper References with link
```
Comment (SanjeevKrishnan) [inline README.md]: Make this heading as "References"

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@paraggarg1102 Changes Required
1. Add Videos folder inside Images and Videos with at least one video.
2. Add CAD images of all parts inside Mechanical Design.
3. Check page alignment in Report. Don't break Team Members into two pages.
4. In Report, Cost Structure should be in proper tabular form.
5. Remove extra "images and videos".
6. Remove extra "SRC". All code files should be in one folder named "src".Code files should be of .ino type if it is an Arduino code(not  .txt)
7. Re-edit README.md file. Use HTML tags to justify the text. `<p align="justify">Content</p>` where content is the text you want to justify.
8. Flowchart is missing in README.md
Refer other project documentations like [Automatic-EyeGlass-Cleaner](https://github.com/marsiitr/Automatic-EyeGlass-Cleaner), [Rube-Goldberg-Machine](https://github.com/marsiitr/Rube-Goldberg-Machine), [Maze-Solving-Bot](https://github.com/marsiitr/Maze-Solving-Robot)

**Comment by DanielLarusso1** [comment]
@SanjeevKrishnan I have made all the necessary changes in the READMe, report and will send a pr to parag who will send a pr to you with other changes.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@paraggarg1102 Changes required
1. Code in src has to be of .ino type as it is an Arduino code.
2. Add "Videos" folder inside "Images and Videos" with either CAD stimulation or Arduino code stimulation.
3. Few CAD Images are present at two locations- "Images and Videos" and "Mechanical Design". Remove it from "Images and Videos". Have only full view CAD Image and Flowchart in Images and Videos/Images

File: README.md [hunk 8]
```diff
@@ -1,2 +1,67 @@
 # Pole-Climbing-Robot
 Srishti 2020
+
+![Final Product](https://github.com/DanielLarusso1/Pole-Climbing-Robot/blob/master/Images%20and%20Videos/Images/Robot%20CAD.jpeg)
```
Comment (SanjeevKrishnan) [inline README.md]: As you have moved Robot CAD.jpeg out of Images folder, update this URL with new URL of Complete Bot Design.jpeg

File: README.md [hunk 9]
```diff
@@ -1,2 +1,67 @@
...
+## Abstract
+<p align="justify">Pole climbing robot utilizes a rack and pinion system to perform vertical motion up and down a pole, with the caterpillar motion of the top plate and bottom plate. The robot can climb up the poles within a certain range of diameters.</p>
+
+## Motivation
+<p align="justify">Electricians and workmen have to climbup an electric pole in case of a malfunction, this leads to several accidents. This can be avoided if a robot can be made to climb the pole and perform specialised functions. As a first step towards that, this robot has been designed to climb a pole. It can also be used for surveillance and fruit harvesting.</p>
+
+## Workflow
+![Workflow](https://github.com/DanielLarusso1/Pole-Climbing-Robot/blob/master/Images%20and%20Videos/Images/Workflow.JPG)
```
Comment (SanjeevKrishnan) [inline README.md]: This image is not rendering. Check it.

File: README.md [hunk 10]
```diff
@@ -1,2 +1,67 @@
...
+ 2. [Nitika Gupta](https://github.com/Nitikagupta16)
+ 3. [Parag Garg](https://github.com/paraggarg1102)
+ 4. [Sarthak Sharma](https://github.com/DanielLarusso1)
+
+## Mentors
+ 1. [Aditya Raj]()
+ 2. [Rishika Chandra](https://github.com/chandrarishika14)
+
+## References
+ 1. Rack and Pinion mechanism: https://www.britannica.com/technology/rack-and-pinion
```
Comment (SanjeevKrishnan) [inline README.md]: Change the syntax. It should be `[Rack and Pinion mechanism](https://www.britannica.com/technology/rack-and-pinion)`
Do this for all references.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@paraggarg1102 Minor Changes Required

**Comment by paraggarg1102** [comment]
@SanjeevKrishnan and @Shubhanshu07  I  have made all the necessary  changes.

File: README.md (hunk 10 above)
Comment (SanjeevKrishnan) [reply README.md]: @paraggarg1102 Do this.

**Comment by paraggarg1102** [comment]
@SanjeevKrishnan  and @Shubhanshu07  I have made all the necessary changes.

**Review by SanjeevKrishnan — APPROVED** [review]
@paraggarg1102 PR Looks fine!
@Shubhanshu07 I suggest for merge.

**Comment by agshubhs** [comment]
@paraggarg1102 PR looks Good to me..

**Pull request merged**